pyteal/ast/tmpl.py

```diff
@@ -28,6 +41,15 @@ def __teal__(self, options: "CompileOptions"):
     def type_of(self):
         return self.type
 
+    @classmethod
+    def session_templates(cls) -> list[str]:
+        return list(cls._session_templates.keys())
+
+    @classmethod
+    def clear_session_templates(cls):
```

Comment:
Though this method is made available, it is never actually used. I'm punting _when_ to refresh `_session_templates` because under normal usage, trying to replace too many template variables won't actually break anything.